moto/__init__.py

```diff
@@ -175,7 +175,7 @@ def f() -> "xray_segment_type":
 mock_quicksight = lazy_load(".quicksight", "mock_quicksight")
 mock_ram = lazy_load(".ram", "mock_ram")
 mock_rds = lazy_load(".rds", "mock_rds")
-mock_rdsdata = lazy_load(".rdsdata", "mock_rdsdata")
+mock_rdsdata = lazy_load(".rdsdata", "mock_rdsdata", boto3_name="rds-data")
```

Comment:
The ServerMode tests are currently failing for the `rds-data` service - is that related to this change?

Reply:
I'll back it out and we'll see. That was just a small thing that I noticed and (thought) would be an easy cleanup, but I probably didn't fully understand it.